RemoveAndUp.ps1

```diff
@@ -0,0 +1,21 @@
+# Nazwa skryptu: RemoveAndUp.ps1
+
+# Ścieżka do katalogu, który chcesz usunąć rekurencyjnie
+$directoryPath = "./data"
+
+# Sprawdź, czy katalog istnieje przed próbą usunięcia
+if (Test-Path $directoryPath) {
+    # Usuń katalog rekurencyjnie
+    Remove-Item -Path $directoryPath -Recurse -Force
+    Write-Host "Katalog $directoryPath został pomyślnie usunięty."
```

Comment:
↑  (English <400 mln ppl> is more universal than Polish <40 mln ppl>, future developers that will work on this code will not understand that line of code)